src/components/CommunityPartners/CommunityPartners.vue

```diff
@@ -0,0 +1,262 @@
+          extraClass: "",
+        },
+        {
+          name: "Intlify",
+          image: "https://github.com/intlify.png",
+          link: "https://intlify.dev/",
+          description:
+            "Intlify is a project that aims to improve Developer Experience in software internationalization. We will aim to improve the Developer Experience of internationalization by providing libraries, frameworks, and tools that break down barriers to internationalization.",
+          extraClass: "rounded-full",
+        },
+        {
+          name: "Code for Japan",
+          image: "/assets/code_for_japan.jpg",
+          link: "https://www.code4japan.org/",
+          description:
+            "Code for Japan is one of the largest communities of civic hackers in Japan. We organize hackathons to improve and promote civic tech and collaborate with local governments and communities.",
+          extraClass: "rounded-full",
+        },
+      ],
+      isModalOpen: false,
+      isModalLoading: false,
+      isModalSuccessful: false,
+      isModalFailed: false,
+      sheetDbBaseUrl: 'https://sheetdb.io/api/v1/',
+      sheetDbConfig: {
+        address: import.meta.env.PUBLIC_SHEETDB_ADDRESS,
+        auth_login: import.meta.env.PUBLIC_SHEETDB_LOGIN,
+        auth_password: import.meta.env.PUBLIC_SHEETDB_PASSWORD,
+      },
+    };
+  },
+  methods: {
+    toggleModal: function () {
+      const htmlEl = document.documentElement;
+      const headerEl = document.querySelector("#app");
+
+      if (this.isModalOpen) {
+        htmlEl.classList.remove("overflow-y-hidden");
+        headerEl.classList.remove("hidden");
+      } else {
+        htmlEl.classList.add("overflow-y-hidden");
+        headerEl.classList.add("hidden");
+      }
+
+      this.isModalOpen = !this.isModalOpen;
+    },
+    fetchSendPostData: async function (url = '', data = {}) {
+      const response = await fetch(url, {
+        method: 'POST',
+        headers: new Headers({
+            'Content-Type': 'application/json',
+            'Authorization': `Basic ${btoa(`${this.sheetDbConfig.auth_login}:${this.sheetDbConfig.auth_password}`)}`
+        }),
+        body: JSON.stringify(data)
+      });
+      if (response.ok) return response.json();
+      throw new Error(response.status);
+    },
+    sheetDBCreateRow: async function(address, data) {
+      return this.fetchSendPostData(this.sheetDbBaseUrl + address, {
+        data: { ...data }
+      });
+    },
+    handleFormSubmit: async function ({
+        founderName,
+        communityName,
+        website,
+        email,
+        whyPartnerWithUs,
+        howYouHearAboutUs
+    }) {
+      this.isModalLoading = true;
+
+      this.sheetDBCreateRow(this.sheetDbConfig.address, {
+          'Founder Name': founderName,
+          'Community Name': communityName,
+          'Website': website,
+          'Email': email,
+          'Why do you want to be partner with us?': whyPartnerWithUs,
```

Comment:
```suggestion
          'Why do you want to partner with us?': whyPartnerWithUs,
```

Reply:
@vinzvinci You need to change the header column in the google forms according to this one.

From:
![image](https://user-images.githubusercontent.com/28738855/146866071-877ee614-3abc-4ed8-a87a-37624d8ec72b.png)

To:
![image](https://user-images.githubusercontent.com/28738855/146865930-b4cf008c-5439-46b9-bb92-25194eddd88a.png)

Let me know once you read this before I commit so we can avoid bugs.

Reply:
Hey I remove that **be** on the spreadsheet